src/IonSpan.ts

```diff
@@ -51,6 +51,14 @@ export abstract class Span {
 
   protected abstract clone(start: number, len: number): Span;
 
+  locator(pos: number) : number[] {
+    throw new Error("not implemented");
```

Comment:
Well, I really just want these methods implemented on text spans, but for some reason the text parser constructor uses the abstract 'Span' class.  This seems wrong -- is there ever a reason to pass a binary span to a text reader?   If we can change the IonParserTextRaw constructor to take a TextSpan, I can clean this up easily enough....